examples/demo/landing_page_demo_2.tut

```diff
@@ -4,9 +4,11 @@ source config.sh.in
 
 run() {
     c PyFunceble --version
-    c PyFunceble -d 192.168.178.60 # TODO doesn't print the status
-    c PyFunceble -d 192.168.178.250 # TODO doesn't print the status
-    c PyFunceble -d 192.168.178.259
+    c PyFunceble -d 9.9.9.9                         # Neutral stable
```

Comment:
Would actually rather suggest to to do https://github.com/funilrys/PyFunceble/pull/262/files/aa75d8886d6196b4dead17b815e04e6908b6306b..f7430a448961eb63856e3bc00f358d9b9176bc2f#r739692268 here as. 2 IP to same service.

`c PyFunceble -d 9.9.9.9 149.112.112.112`

Reply:
@spirillen I think it's a good idea to show two domains at once, but I don't like using this as the very first example. I like https://github.com/funilrys/PyFunceble/pull/262#discussion_r739692268 better.

Reply:
Hmm, my second thought was based on the stability on the quad9's IP addresses, how about reordering them then?

(The comment line would also be kept shorter :wink: )  Just throwing some ball's into the air here.

Reply:
I think it's best to keep a single IP as the first example, especially `9.9.9.9` because it's so simple. And I think that it's better that two IPs in a single command have different statuses, i.e. one of them is active and the other inactive - makes it more noticable IMO. But these are just nitpicks, I don't feel as strongly about this as I may have made it seem.

Reply:
I buy your argument with two diff result status... it's not bad...

Reply:
@funilrys Your thoughts?

Reply:
Hi there, single IP as the first example is good for me. 

Later, when the generation and upload to the gh-pages are automated, we could document all possible cases @spirillen.

Reply:
I'm open for making further documentation, there are just the "just", if you remember I was kind of starting making examples for various use cases (which I then deleted again before it got merged into 4.0) as making all that inside the documentation became more messy than good constructive use examples.

I can't help thinking on @T145 comment in https://github.com/funilrys/PyFunceble/issues/265#issuecomment-962494029, not that I'll dare to say I fully understand it yet, but the is obviously room for some rethinking. Please do throw anything into the pot :cauldron:

![image](https://user-images.githubusercontent.com/44526987/140663254-8e47653c-2a31-4ed0-8201-e4638a53634c.png)

Non the less as always, please make all docs related issues over there... (NOT :us: but https://mypdns.org/spirillen/PyFunceble) so i'm in control of the related issues work-boards etc. 

PS: sorry for typos, as I switches between EN and DK keyboard layout.